pennylane/qchem/openfermion_obs.py

```diff
@@ -854,17 +840,7 @@ def molecular_hamiltonian(
     |
 
     Args:
```

Comment:
Should we still mention the old arguments as a `note` or `Usage Details` with a message specifying this is no longer a preferred workflow? 

cc @trbromley

Reply:
Yes please. We can add a note below this args section mentioning the old signature and that it is deprecated, but use an expandable usage details section at the bottom of the page to go into further details.